### ISSUE TITLE
[Documentation] trollop to optimist guide / FAQ?

### ISSUE THREAD

**Issue by rubyFeedback** [ManageIQ/optimist#141]
I inherited another project that requires trollop. I am willing to move to optimist,
but I would love to have some guidelines as to what to do in order to move. The
code base is foreign to me, including the style - very alien, so I'd like to have 
some document that can help me in the process of porting things.

For instance:

    opts = Trollop.with_standard_exception_handling(trollop_parser) {
    trollop_parser.parse(i)

Can such method names be just used as a 1:1 replacement? If not what else
can be used? Could some document be added (can be short and concise,
just a tiny helper tool) and then linked to it from the main README perhaps?

**Comment by kbrock** [comment]
Hello @rubyFeedback 

Depending upon the version of trollop that you use, it should be a search and replace.

**Comment by Fryguy** [comment]
Agreed with @kbrock  - It should be a simple find `Trollop` replace with `Optimist`
